bloom-housing/bloom
# 2172/visual design bugs

# Pull Request Template

## Issue Overview

This PR addresses #2172 #2760


## Description

Fixes visual issues:
- added modal backdrop opacity 50%
- updated checkbox input font weight to normal
- removed features section right padding
- normalized radio input spacing

## How Can This Be Tested/Reviewed?

Just check screenshots from the issue and make sure that the problems have been fixed.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
